subprojects/gdk_rust/gdk_electrum/src/lib.rs

```diff
@@ -112,6 +112,7 @@ impl Closer {
 
 pub struct ElectrumSession {
     pub data_root: String,
+    pub proxy: String,
```

Comment:
I personally prefer the explicitness of an `Option` over using an empty string to indicate the proxy is off. Up to you.

Reply:
Yes, an option is better, done